test/server/listener_manager_impl_test.cc

```diff
@@ -337,7 +337,7 @@ class NonTerminalFilterFactory : public Configuration::NamedNetworkFilterConfigF
   }
 
   ProtobufTypes::MessagePtr createEmptyConfigProto() override {
-    return std::make_unique<Envoy::ProtobufWkt::Empty>();
+    return std::make_unique<Envoy::ProtobufWkt::Struct>();
```

Comment:
How come this?

Reply:
These are all test filters and they break the assertion that bans empty as config proto. It's never used so changing the type doesn't cause an issue. It seems unnecessary to create empty proto for every fake filter.

Reply:
I suggest comments or some common test proto type to avoid implicit knowledge here.

Reply:
Added comments.